packages/opentelemetry-api-metrics/README.md

```diff
@@ -0,0 +1,108 @@
+# OpenTelemetry API for JavaScript
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This package provides everything needed to interact with the OpenTelemetry Metrics API, including all TypeScript interfaces, enums, and no-op implementations. It is intended for use both on the server and in the browser.
```

Comment:
if i understood correctly this package will be merged at some point when the metrics specs are froozen right ? If so could we add some explaination so user expect to be able to use this only temporarily

Reply:
7b81421b338dfa442929d0d957e6cf84265cb87d